OpenMEEG/src/geometry.cpp

```diff
@@ -385,6 +385,26 @@ namespace OpenMEEG {
         return OK;
     }
 
+    bool Geometry::check(const Matrix& mat) const {
+        bool OK = true;
+        if (!is_nested()) {
+            std::cerr << "Dipoles are only allowed when geometry is nested." << std::endl;
```

Comment:
IMO, we can have dipole anywhere. It should not be restricted to the inner_volume.
I remember some canadians who wanted to add dipoles in the scalp at eye location to model the eye blinks.
(but you could say `check_inner()` and later we'll extend this function)

Reply:
I like the idea of the check_inner. It's explicit.

@eolivi can you push directly to this PR if you have a bit of time?